packages/debug/src/common/debug-common.ts

```diff
@@ -173,6 +178,7 @@ export class DebugSessionStateAccumulator implements DebugSessionState {
     private _allThreadsStopped: boolean | undefined;
     private _stoppedThreads = new Set<number>();
     private _breakpoints = new Map<string, DebugProtocol.Breakpoint>();
+    private _capabilities: DebugProtocol.Capabilities = {};
```

Comment:
Please do not use `_` for private fields. From https://github.com/theia-ide/theia/wiki/Coding-Guidelines:
`Do not use "_" as a prefix for private properties`